droidmediacamera2.cpp

```diff
@@ -1783,9 +1811,11 @@ char *droid_media_camera_get_parameters(DroidMediaCamera *camera)
                  params += "video-stabilization-supported = true;";
             }
             break;
-        case ACAMERA_CONTROL_ZOOM_RATIO_RANGE:
-            params += "max-zoom="+std::to_string(entry.data.f[1])+";";
-            break;
+#if ANDROID_MAJOR >= 9 
+	case ACAMERA_CONTROL_ZOOM_RATIO_RANGE:
+           params += "max-zoom="+std::to_string(entry.data.f[1])+";";
+           break;
```

Comment:
Should we return max-zoom also for older devices?

Reply:
I think the parameter means the max zoom of optical zooms. This does not really make sense if zoom is only done by cropping the image
>  	Minimum and maximum zoom ratios supported by this camera device.
https://developer.android.google.cn/ndk/reference/group/camera#group___camera_1gga49cf3e5a3deefe079ad036a8fac14627af00bd3268478899b538e6ae1323720fc